[P0] 清理物件分類右鍵選單中的新增群組假入口

## 背景
Tables / Views / Queries 物件分類右鍵選單仍顯示 `新增群組`，但實際只會回覆尚未支援。這種假入口容易讓使用者誤以為功能可用，應先清理 UI 行為。

## Todo
- 盤點 Tables / Views / Queries 分類節點的右鍵選單。
- 移除 `新增群組` 假入口，或改成明確不可用文案與 disabled 狀態。
- 若保留提示，文案需清楚說明目前尚未支援，不應像可執行功能。
- 若 README / Roadmap 有群組功能描述，需同步標示尚未完成。

## 驗收
- 建置通過
- 對應 smoke test 通過
- UI/DB 行為符合描述
- Tables / Views / Queries 右鍵選單不再出現會誤導的可點擊假入口。

## 參考
- `mySQLPunk/Form1.cs`
- `ShowGroupUnavailable`

已處理 #13：移除 Tables / Views / Queries 分類節點右鍵選單中的『新增群組』假入口，避免誤導使用者。

- 調整： 移除三個分類節點（Tables / Views / Queries）context menu 的『新增群組』項目（原本只會呼叫 ）。
- 文件： 釐清『群組功能』僅涵蓋『連線群組』，並把『Tables 群組』文案改成『Tables 節點』避免混淆。
- 測試：此環境為 Linux，缺少 .NET Framework v4.7.2 targeting pack，MSBuild version 17.8.49+7806cbf7b for .NET
/usr/lib/dotnet/sdk/8.0.126/Microsoft.Common.CurrentVersion.targets(1241,5): error MSB3644: The reference assemblies for .NETFramework,Version=v4.7.2 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks [/data/yangchen/home/projects/mySQLPunk/mySQLPunk/mySQLPunk.csproj] 會出現 MSB3644；Mono 
>>>> xbuild tool is deprecated and will be removed in future updates, use msbuild instead <<<<

XBuild Engine Version 14.0
Mono, Version 6.8.0.105
Copyright (C) 2005-2013 Various Mono authors

Build started 05/17/2026 13:50:18.
__________________________________________________
Project "/data/yangchen/home/projects/mySQLPunk/mySQLPunk.sln" (default target(s)):
	Target ValidateSolutionConfiguration:
		Building solution configuration "Debug|Any CPU".
	Target Build:
		Project "/data/yangchen/home/projects/mySQLPunk/mySQLPunk/mySQLPunk.csproj" (default target(s)):
			Target PrepareForBuild:
				Configuration: Debug Platform: AnyCPU
			Target GetFrameworkPaths:
/usr/lib/mono/xbuild/14.0/bin/Microsoft.Common.targets:  warning : TargetFrameworkVersion 'v4.7.2' not supported by this toolset (ToolsVersion: 14.0).
: error : Error building target _AddValueTupleFrameworkDirectory: Method 'NormalizeDirectory(/data/yangchen/home/projects/mySQLPunk/packages/System.ValueTuple.4.6.2/build/net471/, .., .., lib, net47)' arguments cannot be evaluated'
		Done building project "/data/yangchen/home/projects/mySQLPunk/mySQLPunk/mySQLPunk.csproj".-- FAILED
	Task "MSBuild" execution -- FAILED
	Done building target "Build" in project "/data/yangchen/home/projects/mySQLPunk/mySQLPunk.sln".-- FAILED
Done building project "/data/yangchen/home/projects/mySQLPunk/mySQLPunk.sln".-- FAILED

Build FAILED.

Warnings:

/data/yangchen/home/projects/mySQLPunk/mySQLPunk.sln (default targets) ->
(Build target) ->
/data/yangchen/home/projects/mySQLPunk/mySQLPunk/mySQLPunk.csproj (default targets) ->
/usr/lib/mono/xbuild/14.0/bin/Microsoft.Common.targets (GetFrameworkPaths target) ->

	/usr/lib/mono/xbuild/14.0/bin/Microsoft.Common.targets:  warning : TargetFrameworkVersion 'v4.7.2' not supported by this toolset (ToolsVersion: 14.0).

Errors:

/data/yangchen/home/projects/mySQLPunk/mySQLPunk.sln (default targets) ->
(Build target) ->
/data/yangchen/home/projects/mySQLPunk/mySQLPunk/mySQLPunk.csproj (default targets) ->

	: error : Error building target _AddValueTupleFrameworkDirectory: Method 'NormalizeDirectory(/data/yangchen/home/projects/mySQLPunk/packages/System.ValueTuple.4.6.2/build/net471/, .., .., lib, net47)' arguments cannot be evaluated'

	 1 Warning(s)
	 1 Error(s)

Time Elapsed 00:00:00.1210890 也無法處理 v4.7.2，因此未能在這裡完成建置與 UI smoke test。建議在 Windows/VS 用  或直接開啟執行後確認右鍵選單行為。

Commit：effc058 (fix(ui): 移除物件分類選單假入口)

--- a/mySQLPunk/Form1.cs
+++ b/mySQLPunk/Form1.cs
@@ -6762,13 +6762,6 @@ namespace mySQLPunk
             return menu;
         }
 
-        private void ShowGroupUnavailable()
-        {
-            string message = Localization.T("Menu.GroupUnavailable");
-            UpdateMainStatus(message);
-            MessageBox.Show(message, Localization.T("Menu.NewGroup"), MessageBoxButtons.OK, MessageBoxIcon.Information);
-        }
-
         private void RefreshConnectionList()
         {
             drawLists();

--- a/mySQLPunk/Localization.cs
+++ b/mySQLPunk/Localization.cs
@@ -58,7 +58,6 @@ namespace mySQLPunk
             Add("Menu.ColorBlue", "藍色", "Blue");
             Add("Menu.ColorPurple", "紫色", "Purple");
             Add("Menu.ManageGroups", "管理群組", "Manage Groups");
-            Add("Menu.GroupUnavailable", "尚未建立群組功能", "Groups are not available yet");
             Add("Menu.GroupName", "群組名稱", "Group Name");
             Add("Menu.GroupNamePrompt", "請輸入群組名稱：", "Enter a group name:");
             Add("Menu.MoveToGroup", "移至群組...", "Move to Group...");

